go.mod

```diff
@@ -542,3 +541,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.6.0 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+replace github.com/docker/docker => github.com/docker/docker v28.1.1+incompatible
```

Comment:
Why is this needed?

Reply:
We'll just wait for the next trivy release with the docker bump.
This is not solving anything anyway. I don't see docker in the PRs opened by dependabot.

Reply:
trivy wants this version v28.1.1+incompatible

go-containerregistry wants to move past it.